Specification.md

```diff
@@ -172,18 +182,40 @@ but what is allowed by this specification.
 There MUST appear one “blank line” between subsequent *records*;
 additional “blank lines” MAY appear.
 
-The *records* don’t have to appear in any order.
-There MAY exist multiple *records* for the same day.
-These are treated as distinct.
-
-### Technical remarks
+*Records* MAY appear in any order in the file.
 
-The file extension MUST be `.klg`, e.g. `times.klg`.
+There MAY exist multiple *records* with the same date.
+These MUST be treated as distinct.
```

Comment:
I’m actually not sure whether this addendum is super worthwhile at all – or well-written for that matter. What I tried to say is that an evaluating program shouldn’t combine the records only because they are at the same date. So for example if you have `myfile.klg` that contains three records all at the same date, the CLI would report:

```
$ klog total myfile.klg
Total: 20h
(In 3 records)
```

So on the one hand the total times are added up, but it should just count the number of records naively. Another case where this is important is when filtering records by tags. E.g., consider the following file:

```
2020-01-01
#foo
  3h

2020-01-01
  2h
```

If you filter for `#foo`, it should report the total time as `3h`, not `5h`. In other words, the above data is not equal to this:

```
2020-01-01
#foo
  3h
  2h
```

Reply:
I moved this part down to the “Total Time” section, because I think it’s a practical aspect. Does that make sense to you? https://github.com/jotaen/klog/pull/88/commits/f91af9ac5698054576ef5eba0e95f51adb88e386

Reply:
I think that makes total sense. The way it's phrased within the parentheses feel a bit "un-academic", but makes the point very clear.

Reply:
Haha, this is exactly what I thought. I have removed that parenthesized sentence again, I think the instruction is clear without example. (https://github.com/jotaen/klog/pull/88/commits/3e0999d8f95996e322b83ac231985a269caac859)